### DISCUSSION TITLE
Controls?

### DISCUSSION THREAD

**Discussion by TheZoidMaster** [TodePond/CellPond#300] [Help] [answered]
What exactly are the controls for this? It looks like fun, but I can't get the controls right.

**Comment by loglot** [comment, marked as answer]
left click to place many
middle click to place one
right click to pick color

left click out of bounds to make grid 1x1 with current color, click again to fully reset grid
right click out of bounds to change color through presets

ctrl s to save rules to a file
ctrl c to save rules to clipboard
ctrl v to load rules from clipboard

scroll wheel to zoom in and out
ctrl scroll wheel to zoom in and out of rules
alt scroll wheel to scroll through rules

**Reply by loglot** [reply]
there may be more, but this is what i can think of currently

**Reply by loafhouse** [reply]
right click and move mouse to move cell thing
ctrl-o to open a file (or just drop it)
0-6 can set the amount of iterations of that generation happen.
"+" enables loop-around
 "-" disables loop-around
shift scroll changes the brush size (https://github.com/TodePond/CellPond/issues/282#issuecomment-1616728059)
